platform/macos/src/MGLMapViewDelegate.h

```diff
@@ -151,6 +151,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)mapViewDidFinishRenderingFrame:(MGLMapView *)mapView fullyRendered:(BOOL)fullyRendered;
 
+/**
+ Tells the delegate that the map view is entering an idle state, and no more
+ drawing will be necessary until new data is loaded or there is some interaction
+ with the map.
+ 
+ - No camera transitions are in progress
+ - All currently requested tiles have loaded
+ - All fade/transition animations have completed
+ 
+ @param mapView The map view that has just entered the idle state.
+ */
+- (void)mapViewDidEnterIdle:(MGLMapView *)mapView;
```

Comment:
“Idle” isn’t a noun, so “entering idle” sounds a bit awkward. `-mapViewDidBecomeIdle:` sounds better.

Reply:
@ChrisLoer `-mapViewDidBeginIdling` could be another option 🤷‍♂️

Reply:
Changed to `didBecomeIdle`